### PR TITLE
new recipe: kealib/1.4.13

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **kealib/1.4.13**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by jgsogo — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

File: recipes/kealib/all/patches/fix-export-symbols-and-cmake.patch [hunk 1]
```diff
@@ -0,0 +1,216 @@
+--- a/CMakeLists.txt
```
Comment (madebr) [inline recipes/kealib/all/patches/fix-export-symbols-and-cmake.patch:1]: @SpaceIm 
Have you sent this patch upstream?

Comment (SpaceIm) [reply recipes/kealib/all/patches/fix-export-symbols-and-cmake.patch:1]: Not yet. I've opened an issue: https://github.com/ubarsc/kealib/issues/7, and commented first draft from library author https://github.com/ubarsc/kealib/compare/exportheader
But yes I should open a PR with this patch.

Comment (SpaceIm) [reply recipes/kealib/all/patches/fix-export-symbols-and-cmake.patch:1]: https://github.com/ubarsc/kealib/pull/10

File: recipes/kealib/all/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.5)
```
Comment (intelligide) [inline recipes/kealib/all/CMakeLists.txt:1]: Why 3.5 ?

Comment (SpaceIm) [reply recipes/kealib/all/CMakeLists.txt:1]: Because upstream CMakeLists requires 3.5.

**Review by danimtb — APPROVED** [review]

**Pull request merged**